openedx/core/djangoapps/demographics/models.py

```diff
@@ -0,0 +1,20 @@
+from django.contrib.auth import get_user_model
+from django.db import models
+from model_utils.models import TimeStampedModel
+from simple_history.models import HistoricalRecords
+
+User = get_user_model()
+
+
+class UserDemographics(TimeStampedModel):
+    """
+    A Users Demographics platform related data in support of the Demographics
+    IDA and features
```

Comment:
I assume this app plays well with Open edX, which doesn't have the IDA?

Reply:
There is nothing in here that will block open edx users.  We should discuss whether or not this should go into a plugin later.